test/__snapshots__/terserOptions-option.test.js.snap.webpack4

```diff
@@ -662,9 +662,4 @@ Object {
 
 exports[`terserOptions option should match snapshot for the "warnings" option with the "true" value: errors 1`] = `Array []`;
 
-exports[`terserOptions option should match snapshot for the "warnings" option with the "true" value: warnings 1`] = `
-Array [
-  "Terser Plugin: Dropping unreachable code [main.js:91,41]",
-  "Terser Plugin: Dropping unused function foo [main.js:91,9]",
-]
-`;
+exports[`terserOptions option should match snapshot for the "warnings" option with the "true" value: warnings 1`] = `Array []`;
```

Comment:
I'm not sure why these arrays are empty now 🤔

Reply:
warnings are deprecated https://github.com/terser/terser/commit/71dc3456be968ddce2281b93beb4079e609e28ef

Reply:
It is very bad, very